src/screens/SUE/SueListScreen.tsx

```diff
@@ -53,6 +55,22 @@ export const SueListScreen = ({ navigation, route }: Props) => {
       })
   );
 
+  const { data: servicesData } = useQuery([QUERY_TYPES.SUE.SERVICES, {}], () =>
+    getQuery(QUERY_TYPES.SUE.SERVICES)({})
+  );
+
+  const servicesFilterData = useMemo(() => {
```

Comment:
what about naming this `services` like we have `statuses`?

Reply:
Yeah, that makes sense. 👍